FMDN upstream part 9 (final): new FMDN content for the Fast Pair service documentation (#15729)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

* :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN` - The option enables the FMDN extension.

  * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT` - The option enables the Detecting Unwanted Location Trackers (DULT) support in the FMDN extension (see :ref:`ug_bt_fast_pair_prerequisite_ops_fmdn_dult_integration`):

    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MANUFACTURER_NAME` - The option configures the manufacturer name parameter.
    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MODEL_NAME` - The option configures the model name parameter.
    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_ACCESSORY_CATEGORY` - The option configures the accessory category parameter.
    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_MAJOR`, :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_MINOR` and :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_REVISION` - These options configure the firmware version parameter.

  * There are following advertising configuration options for the FMDN extension (see :ref:`ug_bt_fast_pair_advertising_fmdn`):

>     * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_TX_POWER` - The option configures the TX power in dBm units set in Bluetooth LE controller for FMDN advertising and connections.

Reviewer: ```suggestion
    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_TX_POWER` - The option sets the TX power (dBm) in the Bluetooth LE controller for FMDN advertising and connections.
```

    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_ECC_SECP160R1` and :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_ECC_SECP256R1` - These options are used to select the elliptic curve for calculating the FMDN advertising payload.
      The secp160r1 elliptic curve is enabled by default.

  * There are following battery configuration options for the FMDN extension (see :ref:`ug_bt_fast_pair_advertising_fmdn_battery` and :ref:`ug_bt_fast_pair_gatt_service_fmdn_battery_dult`):

    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_BATTERY_LEVEL_LOW_THR` - The option configures the threshold percentage value for entering the low battery state as defined in the FMDN extension.
    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_BATTERY_LEVEL_CRITICAL_THR` - The option configures the threshold percentage value for entering the critically low battery state as defined in the FMDN extension.
    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_BATTERY_DULT` - The option configures the FMDN module to pass the battery information to the DULT module and to support its mechanism for providing battery information to the connected peers.

>       This option can only be used when the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT` is enabled.

Reviewer: ```suggestion
      This option can only be used when the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT` Kconfig option is enabled.
```

* Make sure that your advertising session is shorter than the value in the :kconfig:option:`CONFIG_BT_RPA_TIMEOUT` option.
* Call the :c:func:`bt_le_oob_get_local` function to trigger RPA rotation and reset the RPA timeout right before advertising starts.

.. note::

>    If you use the FMDN extension, and your Provider is provisioned as an FMDN beacon, you must not use the :c:func:`bt_le_oob_get_local` function.

Reviewer: ```suggestion
   If you use the FMDN extension, and your Provider is provisioned as an FMDN beacon, do not use the :c:func:`bt_le_oob_get_local` function.
```
